### PR TITLE
Make nginx default

### PR DESCRIPTION
**Pull request by antonmolodykh**
Refs #689

### CODE REVIEW

File: ansible/roles/enable-nginx/files/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  server_tokens off;
+  listen 80 default_server;
+
+  location / {
+    client_max_body_size 4G;
+    proxy_pass http://ose;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_set_header X-Forwarded-Protocol https;
```
Comment (vpetersson) [inline ansible/roles/enable-nginx/files/nginx.conf]: This is wrong.

File: ansible/site.yml [hunk 2]
```diff
@@ -30,4 +30,6 @@
     - rpi-update
     - screenly
     - splashscreen
+    - enable-nginx
```
Comment (vpetersson) [inline ansible/site.yml]: Feels a bit award to have both. Let's merge them into one role called `nginx` and mark it up properly with comments.

**Comment by vpetersson** [comment]
How about Swagger? I didn't see any changes related to it, which makes me think it will not work.

**Comment by antonmolodykh** [comment]
You're right, I did not take one case into account. Fixed.

I don't see a good way to remove the `listen` variable from the configurations. at least for the reason that from there we get the ip on which we start the server.

File: ansible/roles/nginx/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    - disable-nginx
+
+- name: Remove nginx config
+  file:
+    path:  /etc/nginx/sites-enabled/screenly.conf
+    state: absent
+  tags:
+    - disable-nginx
+
+- name: Remove self-signed certificates
```
Comment (vpetersson) [inline ansible/roles/nginx/tasks/main.yml]: Let's leave these actually. No point in regenerating them if this is recreated later.

**Comment by vpetersson** [comment]
LGTM.

**Pull request merged**